pyproject.toml

```diff
@@ -25,6 +25,8 @@ requires-python = ">=3.11"
 dependencies = [
     "arches>=8.1.0a0,<8.2.0",
     "arches-component-lab @ git+https://github.com/archesproject/arches-component-lab@main",
+    "arches-controlled-lists @ git+https://github.com/archesproject/arches-controlled-lists@main",
```

Comment:
Is there a need for this to depend on main vs stable/released?  Same q for querysets.